src/gui/qgsdatabasecommentdialog.h

```diff
@@ -0,0 +1,50 @@
+/***************************************************************************
+    qgsdatabasecommentdialog.h
+    ---------------------
+    begin                : September 2025
+    copyright            : (C) 2025 by Jan Caha
+    email                : jan.caha at outlook dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSDATABASECOMMENTDIALOG_H
+#define QGSDATABASECOMMENTDIALOG_H
+
+#include <QDialog>
+#include <QDialogButtonBox>
+#include <QPlainTextEdit>
+
+#include "qgis_gui.h"
+/**
+ * \ingroup gui
+ * \brief Dialog which displays comment for a specific database item.
```

Comment:
Actually there's nothing database specific about this dialog. Can you rename the files, the class, and the documentation to something generic? E.g. `QgsCommentInputDialog`?